packages/zero-pg/package.json

```diff
@@ -18,6 +18,8 @@
   },
   "prettier": "@rocicorp/prettier-config",
   "dependencies": {
+    "@databases/escape-identifier": "^1.0.3",
```

Comment:
Changes to a package.json usually comes with changes to package-lock.json too. Did you forget to include that?